custom/ewsghana/api.py

```diff
@@ -855,15 +855,18 @@ def _report_notfication_sync(self, report, interval, day):
             return
 
         notifications = ReportNotification.by_domain_and_owner(self.domain, user.get_id)
-        reports = []
         for n in notifications:
-            for config_id in n.config_ids:
-                config = ReportConfig.get(config_id)
-                reports.append((config.filters.get('location_id'), config.report_slug, interval))
-
-        if report.report not in self.REPORT_MAP or (location.location_id, self.REPORT_MAP[report.report],
-                                                    interval) in reports:
-            return
+            if len(n.config_ids) == 1:
+                # Migrated reports have only one config
+                config = ReportConfig.get(doc_id=n.config_ids[0])
+                location_id = config.filters.get('location_id')
+                slug = self.REPORT_MAP[report.report]
+                report_slug = config.report_slug
+                if (n.day, location_id, report_slug, n.interval) == (day, location.location_id, slug, interval):
+                    if not n.send_to_owner and not n.recipient_emails:
```

Comment:
Mike told that some of his reports wasn't migrated at all. I thought they wasn't migrated because he has reports for archived location but it's not true since we didn't ignore archived locations before.